ipv-stub/src/services/dynamodb-form-response-service.ts

```diff
@@ -43,6 +44,23 @@ export const putReverificationWithAccessToken = async (
   });
 };
 
+export const getReverificationWithAccessToken = async (
+  accessToken: string
+): Promise<string | undefined> => {
+  const result = await dynamo.get({
+    TableName: tableName,
+    Key: {
+      ReverificationId: [accessTokenPrefix, accessToken].join("-"),
+    },
+  });
+
+  if (!result.Item) {
```

Comment:
Would this be achieved just by returning `result.Item?.reverification` at the end, and cutting out this if?